alerts_backend/python/app.py

```diff
@@ -23,40 +24,66 @@
 engine = create_engine(
     "sqlite+pysqlite:////var/db/aeroapi_alerts/aeroapi_alerts.db", echo=False, future=True
 )
+table_name = "aeroapi_alerts_table"
 
 
-def insert_into_db(data_to_insert: Dict[str, Union[str, int]]) -> int:
+@app.before_first_request
```

Comment:
Wow this is really cool, didn't know it existed. I'd recommend just doing this on server launch, though (like at the module level). Wouldn't want to wait until just before the first request to find out there's something wrong with the db.